### PR TITLE
Add"variables" to single collection informations

### PR DESCRIPTION
**Pull request by cettina-tosto**

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf-projects/cads-catalogue-api-service?pullRequest=59) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf-projects/cads-catalogue-api-service?pullRequest=59) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf-projects/cads-catalogue-api-service?pullRequest=59) it.</sub>

File: cads_catalogue_api_service/client.py [hunk 1]
```diff
@@ -362,6 +362,7 @@ def collection_serializer(
     db_model: cads_catalogue.database.Resource,
     request: fastapi.Request,
     preview: bool = False,
+    get_all: bool = True,
```
Comment (keul) [inline cads_catalogue_api_service/client.py:365]: @cettina-tosto why this `get_all` is needed? We already have `preview` for this I think

Comment (cettina-tosto) [reply cads_catalogue_api_service/client.py:365]: Oh okok

**Comment by keul** [comment]
After ECMWF discussion: handle of variables requires more internal discussion. VA team already implemented what they need parsing the layout.json file.
For now: no need of additional features

**Pull request closed without merging**